### DISCUSSION TITLE
[Question] Suction gripper in Isaac Lab

### DISCUSSION THREAD

**Discussion by chungshan** [isaac-sim/IsaacLab#712] [Q&A]
In Isaac Sim, it can simulate suction gripper.
But in Isaac Lab, how to add suction gripper?

**Comment by SonYeongGwang** [comment]
Did you find the way?

**Comment by TerenceXue-tech** [comment]
I find the source "UR10_with_suction" at omniverse://localhost/NVIDIA/Assets/Isaac/4.2/Isaac/Robots/UR10/ur10_short_suction.usd
But I don't know how to implement suction action
